packages/client-assertion-validation/src/utils.ts

```diff
@@ -166,7 +166,7 @@ export const validateDigest = (
   }
   const result = ClientAssertionDigest.safeParse(digest);
   if (!result.success) {
-    return failedValidation([digestClaimNotFound(result.error.toString())]);
+    return failedValidation([invalidDigestClaim(JSON.stringify(result.error))]);
```

Comment:
```suggestion
    return failedValidation([invalidClaim(result.error)]);
```
We could also leverage the already defined `invalidClaim` which has a specific parsing for zodErrors but it needs `failedValidation` to be extended with `CommonErrorCodes` too